Sources/Client/State/PopupState.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import SwiftUI
+
+
+// MARK: - PopupObject
+
+
+struct PopupObject {
+  let title: String
+  let subtitle: String
+  private(set) var image: Image? = nil
```

Comment:
why private(set) var instead of let? i can't see any reason

Reply:
did it in order to avoid having to write a custom initialiser, but I guess that conceptually it's better to have it declared as a public let even though that means writing a few extra lines of code

On it 👍